initial crack at hover text

lindsayplatt opened the pull request:
@ldecicco-USGS here's what I've got so far. The mouseover directions show up in the svg object, but I can't seem to actually see anything when I open it in a browser. I feel like there is css or something missing.

Commit:
initial crack at hover text

lindsaycarr commented:
@jread-usgs you might be able to notice an obvious issue

jordansread commented on scripts/visualize/visualize_map.R:
```diff
@@ -58,11 +58,22 @@ visualize.states_svg <- function(viz){
     xml_add_child(xml_add_child(gf, 'g', transform=transform), 
-                  'use', 'xlink:href'=paste0("#", id.use), id=id.name, class='state-foreground')
+                  'use', 'xlink:href'=paste0("#", id.use), id=id.name, class='state-foreground',
+                  onmousemove=sprintf("hovertext('%s',evt);", state.name[i]),
```
we are going to want to do something w/ a string replace since those state names are all lowercase. Should be fine for now.

jordansread commented on scripts/visualize/visualize_map.R:
```diff
@@ -58,11 +58,22 @@ visualize.states_svg <- function(viz){
+  
+  g.tool <- xml_add_child(svg,'g',id='tooltip-group')
+  xml_add_child(g.tool, 'rect', id="tooltip-box", height="24", class="tooltip-box")
+  xml_add_child(g.tool, 'use', id="tooltip-point", d="M-6,-11 l6,10 l6,-11", class="tooltip-box")
```
These dimensions are specific to the size of that other svg, so we will need to tweak this a bit.

lindsayplatt replied:
So what do the dimensions mean?

jread-usgs commented:
@lindsaycarr does this work when you build the viz?

jread-usgs commented:
@lindsaycarr can you resolve the conflicts?

lindsaycarr commented:
Works as in it does not break anything. Can't see the hovers though

Commit:
merge conflicts

Merge remote-tracking branch 'upstream/master'

Conflicts:
	layout/css/map-svg.css
	scripts/visualize/visualize_map.R

jread-usgs commented:
@lindsaycarr I'll pull it down and look and then you review my changes to see what I modify?

lindsaycarr commented:
sounds good

Commit:
fixes for tooltips

Commit:
Merge pull request #1 from jread-usgs/master

fixes for tooltips

lindsaycarr commented:
Jordan's tweaks make it work! https://github.com/lindsaycarr/water-use/pull/1
